packages/web/src/scss/components/Dropdown/README.md

```diff
@@ -1,30 +1,153 @@
+      </svg>
+      <span>More links</span>
+    </a>
+    <a href="#" class="d-flex mb-400">
+      <svg width="24" height="24" aria-hidden="true" class="mr-400">
+        <use xlink:href="/icons/svg/sprite.svg#profile" />
+      </svg>
+      <span>Profile</span>
+    </a>
+    <a href="#" class="d-flex">
+      <svg width="24" height="24" aria-hidden="true" class="mr-400">
+        <use xlink:href="/icons/svg/sprite.svg#help" />
+      </svg>
+      <span>Help</span>
+    </a>
+  </div>
 </div>
 ```
+
+## Advanced usage
+
+```html
+<div class="DropdownWrapper">
+  <button
+    type="button"
+    data-toggle="dropdown"
+    data-target="#DropdownDemo2"
+    data-reference="parent"
+    aria-expanded="false"
+    aria-controls="#DropdownDemo2"
+  >
+    toggle
+  </button>
+  <div class="Dropdown Dropdown--bottom Dropdown--left" id="DropdownDemo2">...</div>
+</div>
+```
+
+## Responsive usage
+
+```html
+<div class="DropdownWrapper">
+  <button
+    type="button"
+    data-toggle="dropdown"
+    data-target="#DropdownDemo2"
+    data-reference="parent"
+    aria-expanded="false"
+    aria-controls="#DropdownDemo2"
+  >
+    toggle
+  </button>
+  <div class="Dropdown Dropdown--bottom Dropdown--left" id="DropdownDemo2" data-breakpoint="tablet">...</div>
+</div>
+```
+
+## Usage with full-width
+
+```html
+<div class="DropdownWrapper">
+  <button
+    type="button"
+    data-toggle="dropdown"
+    data-target="#DropdownDemo2"
+    data-reference="parent"
+    aria-expanded="false"
+    aria-controls="#DropdownDemo2"
+  >
+    toggle
+  </button>
+  <div class="Dropdown Dropdown--bottom Dropdown--left Dropdown--fullWidth" id="DropdownDemo2">...</div>
+</div>
+```
+
+## Align
+
+Dropdown content can be vertically aligned with `.Dropdown--top` or `.Dropdown--bottom` and horizontally `.Dropdown--left` or `.Dropdown--right` side of parent.
+
+## JavaScript
+
+There are two options here. Use the trigger element as an anchor or wrap the menu together with the trigger into a `.DropdownWrapper` class.
+
+| Attribute         | Type                          | Default  | Required | Description                                                                                                    |
+| ----------------- | ----------------------------- | -------- | -------- | -------------------------------------------------------------------------------------------------------------- |
+| `data-target`     | `string`                      | -        | yes      | Target menu element selector                                                                                   |
+| `data-reference`  | `string`                      | `toggle` | no       | Reference selector, it could be 'toggle', 'parent' or Element ID selector                                      |
+| `data-autoWidth`  | `boolean`                     | -        | no       | Expandable content have same with as toggle                                                                    |
```

Comment:
This option has been replaced by the `.Dropdown--fullWidth` class and can be removed (from the TS source too).